docs/sw-build.md

```diff
@@ -0,0 +1,68 @@
+---
+layout: index
+title: sw-build
+navigation_weight: 2
+---
+
+# sw-build
+
+Get a precaching service worker in about ten minutes with the `sw-build` module.
+Simply install the module, then cut and paste one of the provided code samples
+into your build script.
+
+## Install
+
+1. [Install Node.js](https://nodejs.org/en/).
+1. Install the module with NPM.
+
+   ```
+   npm install --save-dev sw-build
+   ```
+
+1. Require `sw-build` in your build script.
+
+   ```
+   const swBuild = require('sw-build');
+   ```
+
+## Generating a Service Worker
+
+### gulp
+
+Add the following to your `gulpfile.js`:
+
+    const swBuild = require('sw-build');
+
+    swBuild.generateSW({
```

Comment:
This code snippet is useful as a generic way of using `sw-build`, but it's not `gulp` specific.

If the code snippet were wrapped in a `gulp.task()` then it would be closer to a `gulp` example.